forms/src/main/java/org/odk/collect/forms/instances/Instance.java

```diff
@@ -27,6 +27,7 @@ public final class Instance {
     // status for instances
     public static final String STATUS_INCOMPLETE = "incomplete";
     public static final String STATUS_INVALID = "invalid";
+    public static final String STATUS_VALID = "valid";
```

Comment:
These statuses are now (even more) confusing. I think we want to rename at least the contestants, but I'd like to hold off until we get a little further into this and are more confident about the lingo we're using to describe each state.

As an aside "incomplete" is now only used when saving before performing finalization (and encryption etc) so would not likely be seen in the wild.

Reply:
It would also come up with instances pushed via adb without the "mark imported instances as finalized" setting, I imagine.